client/pages/_app.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react'
 import '../styles/globals.css'
 import type { AppProps } from 'next/app'
-import { AppProvider } from "../context"
-import { NavBar } from "../components/NavBar"
+import { AppProvider } from '../context'
+import dynamic from 'next/dynamic'
+
+const DynamicNavBar = dynamic(() => import('../components/NavBar').then((mod) => mod.NavBar), {
```

Comment:
What are dynamic imports?

Reply:
When we specify `ssr: false` it let's us load the component on the client-side, it's the only way to get the right initial window size on initial load unfortunately

Reply:
Never seen dynamic used before either; always just handled navbar elements with media queries. This is a cool approach though.